lib/main_common.pm

```diff
@@ -534,8 +545,14 @@ sub load_system_role_tests {
         loadtest "installation/logpackages";
     }
     loadtest "installation/disable_online_repos" if get_var('DISABLE_ONLINE_REPOS') && !get_var('OFFLINE_SUT');
-    loadtest "installation/installer_desktopselection" if is_opensuse;
-    loadtest "installation/system_role" if is_caasp('kubic');
+    if (is_leap) {
```

Comment:
wondering if need here `is_opensource() and !is_staging()` to schedule installer_desktopselection for opensource in general (but with O3 down not able to check it better, just what I remember)

Reply:
That's what we need to clarify with @sysrich. I would either leave it as it is, or introduce some variable which triggers new behavior, like we have done with `SYSTEM_ROLE_FIRST_FLOW`, so we could add it to the medium when required.

Reply:
If the logic is correct I can modify this code to cover tw without the change and have it in more elegant way:
`# ATM changes in the dialog are only in openSUSE staging
    loadtest((is_staging || is_caasp('kubic')) ? "installation/system_role" : "installation/installer_desktopselection") if is_opensuse;`

Reply:
Let's wait for the feedback from @sysrich, I don't see any reason for change as of now.

Reply:
the same change also pending in Leap 15.1 staging, seems we don't need to check ` if (is_leap)` but should rely on an specific variable for new behavior(also commented in another place :)